docs/source/installation.rst

```diff
@@ -162,6 +164,85 @@ memory to execute.
 The ``--gpus all`` flag is necessary to allow Docker to access the required drivers for NVIDIA DALI to work properly.
 
 
+
```

Comment:
add
.. _windows_users:
here (see the similar pattern above the "Docker users" heading)